services/reddit/subreddit-subscribers.service.js

```diff
@@ -0,0 +1,67 @@
+'use strict'
+
+const { BaseJsonService } = require('..')
+const Joi = require('joi')
+const { nonNegativeInteger } = require('../validators')
+const { metric } = require('../text-formatters')
+
+const schema = Joi.object({ subscribers: nonNegativeInteger })
+
+module.exports = class SubredditSubscribers extends BaseJsonService {
+  static get category() {
+    console.log('Category')
+    return 'social'
+  }
+
+  static get route() {
+    console.log('Route')
+    return {
+      base: 'reddit/subreddit-subscribers',
+      pattern: ':subreddit',
+    }
+  }
+
+  static get examples() {
+    return [
+      {
+        title: 'Subreddit followers',
```

Comment:
This should probably also use the `subscribers` verbiage to be consistent with everything else

Reply:
Done, thanks!